.github/workflows/jacktrip.yml

```diff
@@ -342,16 +342,25 @@ jobs:
           # install
           echo "installing Qt"
           make install
+      - name: install openssl for Windows static builds
+        if: matrix.static-qt-version && runner.os == 'Windows'
```

Comment:
I noticed this step still ran when cached qt was used. IIUC this should be the same condition as for `build static Qt on Windows`, right?
```yaml
matrix.static-qt-version && steps.cache-static-qt.outputs.cache-hit != 'true' && runner.os == 'Windows'
```

Reply:
It still needs to build JackTrip with those library files. The build was failing when I included the cache hit requirement.

Reply:
I see, thank you for clarifying. In that case, does it still make sense to have it as an independent install step?

Reply:
probably not 😅

Reply:
Changed back